pkg/metrics/registry.go

```diff
@@ -0,0 +1,15 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	registry = prometheus.NewPedanticRegistry()
+)
+
+// Registerer returns the registerer for metrics that should be exported
+// by the metrics server
+func Registerer() prometheus.Registerer {
```

Comment:
Only MustRegister is used. We could also implement it here:

func MustRegister(cs ...Collector) {
  registry.MustRegister(cs)
}

Reply:
> Only MustRegister is used.

Just coincidence :-) There's a concise interface for registries, `prometheus.Registerer`. Why should I artificially limit it?